### PR TITLE
Remove legacy Node.js support leftovers

### PR DESCRIPTION
**Pull request by bobisjan**
This removes remaining leftovers of legacy Node.js support code #13.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,16 +30,12 @@
     "compression": "^1.6.2",
     "express": "^4.13.3",
     "fastboot": "^2.0.0",
-    "fastboot-express-middleware": "^2.0.0",
-    "fs-promise": "^2.0.3",
```
Comment (bobisjan) [inline package.json:34]: Not used anymore.

File: package.json [hunk 2]
```diff
@@ -30,16 +30,12 @@
...
-    "request": "^2.81.0"
```
Comment (bobisjan) [inline package.json:35]: This is required by `request-promise` for testing only.

**Comment by bobisjan** [comment]
@rwjblue, would you take a look?

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ particular, you can provide a custom:
 
 ## Requirements
 
-FastBoot App Server requires Node.js v4.2 or later.
+FastBoot App Server requires Node.js v8 or later.
```
Comment (rwjblue) [inline README.md:30]: Haha, I'd like to bump this again too but seems good to do separately...

**Review by rwjblue — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (bobisjan) [reply README.md:30]: Yes 😬, #123 is the next step.

**Pull request merged**